src/spox/_value_prop.py

```diff
@@ -24,9 +26,12 @@
 - PropValue -> Optional, Some (has value)
 - None -> Optional, Nothing (no value)
 """
-PropValueType = Union[np.ndarray, list["PropValue"], "PropValue", None]
-ORTValue = Union[np.ndarray, list, None]
-RefValue = Union[np.ndarray, list, float, None]
+PropValueType = Union[
```

Comment:
Don't we get by using the explicit `list` type here? `Sequence` would otherwise be preferable over `Iterable`, too.

Reply:
Updated